zotbins/Waste-Watcher-AWS-IoT-Core
# Andrea fullness

## Description

Added function for ultrasonic sensor to detect and print distance in order to measure fullness of a bin.

## Type of change


## Testing and Verification Instructions

1. Set up sensor in front of surface
2. Start code and check logs for distance measured
3. Increase/decrease distance to validate correct measured distance

## Checklist:
